script/Arcueid/Day 10/10_00_ARC10_3_2.txt

```diff
@@ -798,13 +799,13 @@ I dutifully cast my eyes around the rooftop.
 [sha:032cbc353c72787df69608e165763cb7b484590f]{
 -- Page 221, Offset 24829.
 -- ……確かに。ビルの給水塔の近くに、風を避けるようにテントらしきものが張られていた。
-...Well what do you know.
+...Well, what do you know.
  A small tent has been set up in the lee of the water tower that sits atop the building.
 }
 [sha:6f6185ed30e743776878582bca7a8e8dd98aa8af]{
 -- Page 221, Offset 24830.
 -- この娘、あのテントにいたのか？
-Was this girl inside that tent?
+Is this girl inside that tent?
```

Comment:
?? She's standing right in front of him

Reply:
Misread the context on this one with tired brain